torch/_export/non_strict_utils.py

```diff
@@ -94,7 +94,12 @@ def fakify(
         return t
 
     if not isinstance(t, torch.Tensor):
-        raise ValueError(f"Unsupported input type {type(t)}")
+        raise ValueError(
+            f"Unsupported input type {type(t)}. "
+            "You should use torch.utils._pytree.register_pytree_node/ for custom types if you want to desugar."
```

Comment:
extra `/`

what do you mean "if you want to desugar"...maybe cut?

Reply:
Also it would be cool if you could enumerate out the "..." (I don't actually know what the list is)